packages/dx-react-scheduler/docs/guides/fundamentals.md

```diff
@@ -0,0 +1,44 @@
+# React Scheduler - Fundamentals
+
+The [DevExtreme React Scheduler](https://devexpress.github.io/devextreme-reactive/react/scheduler) allow you to display a structure of appointments and provide a functionality to manage data.
```

Comment:
allow -> allow**s**

Reply:
> display a structure of appointments

Sounds a bit strange)) 

... component that represents/renders/shows/displays appointments?